<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>590</width>
     <height>330</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>彩色图像(&amp;C)</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>通道分离</string>
     </property>
     <addaction name="channel_red"/>
     <addaction name="channel_green"/>
     <addaction name="channel_blue"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="rgb_gray"/>
    <addaction name="adjust_HSL"/>
    <addaction name="adjust_color_levels"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>二值化(&amp;B)</string>
    </property>
    <addaction name="binary_Otsu"/>
    <addaction name="binary_handson"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/menu/new_window.png</normaloff>:/res/menu/new_window.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>新建窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/menu/open.png</normaloff>:/res/menu/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>打开图片</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/tools/close.png</normaloff>:/res/tools/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭(C)</string>
   </property>
   <property name="statusTip">
    <string>关闭图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/menu/save.png</normaloff>:/res/menu/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="statusTip">
    <string>保存图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/menu/save_as.png</normaloff>:/res/menu/save_as.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="statusTip">
    <string>图片另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="statusTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q, Esc</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/menu/undo.png</normaloff>:/res/menu/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="statusTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/menu/redo.png</normaloff>:/res/menu/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="statusTip">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/menu/help.png</normaloff>:/res/menu/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助手册(&amp;H)</string>
   </property>
   <property name="statusTip">
    <string>帮助手册</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="statusTip">
    <string>关于</string>
   </property>
  </action>
  <action name="rgb_gray">
   <property name="text">
    <string>转成灰色</string>
   </property>
  </action>
  <action name="adjust_HSL">
   <property name="text">
    <string>调整HSL</string>
   </property>
  </action>
  <action name="adjust_color_levels">
   <property name="text">
    <string>调整色阶</string>
   </property>
  </action>
  <action name="channel_red">
   <property name="text">
    <string>红色通道</string>
   </property>
  </action>
  <action name="channel_blue">
   <property name="text">
    <string>蓝色通道</string>
   </property>
  </action>
  <action name="channel_green">
   <property name="text">
    <string>绿色通道</string>
   </property>
  </action>
  <action name="binary_Otsu">
   <property name="text">
    <string>Otsu</string>
   </property>
  </action>
  <action name="binary_handson">
   <property name="text">
    <string>手动调节</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
